### PR TITLE
added docs for lookeahead pathfollow2d.xml

### PR DESCRIPTION
**Pull request by pgruenbacher**

### CODE REVIEW

**Comment by pgruenbacher** [comment]
the travis CI details shows everything completed. idk why it's still showing pending....

**Comment by pgruenbacher** [comment]
squashed.

File: doc/classes/PathFollow2D.xml [hunk 1]
```diff
@@ -23,6 +23,7 @@
 			The node's offset along the curve.
 		</member>
 		<member name="lookahead" type="float" setter="set_lookahead" getter="get_lookahead">
+			How far ahead to look ahead of the curve to calculate the tangent if the node is rotating. E.g. shorter lookaheads will lead to faster rotations. Default value: [code]4[/code].
```
Comment (mhilbrunner) [inline doc/classes/PathFollow2D.xml]: I would remove the first "ahead", IMO :) Otherwise looks good to me

**Review by mhilbrunner — DISMISSED** [review]

**Comment by pgruenbacher** [comment]
@mhilbrunner fixed and squashed.

**Comment by akien-mga** [comment]
Thanks!

**Comment by akien-mga** [comment]
Cherry-picked for 3.0.7.

**Pull request merged**